### ISSUE TITLE
It may be time to rename this repo

### ISSUE THREAD

**Issue by michaelgwelch** [metasys-server/basic-services-dotnet#113]
Initially we had planned that this library would only support the basics like our old MMDA did. But it's growing was past that. As such it might make more sense to rename the repo following the release naming: metasys-rest-api-client or some such thing.

Maybe less disruptive to any consumers would be to update the README that this is being archived and add a link to the new repo. Then archive this repo. Then push the source to the new repo and continue development there.

Just helpful I think if the deliverables match the repo name and if the repo name matches what we are actually delivering. Thoughts @Federico-Artioli ?

**Comment by Federico-Artioli** [comment]
Hi Michael, I see your point and to be fair I'm a bit concern to rename now the repo (and everything related as Nuget package etc..). Especially if there is the risk to create some 'confusion' to the current users (maybe they cannot see benefit on rename it). In any case if you think it is really needed I would propose to postpone it a bit due to I'm going to make a new release (v4.2) by mid October. Would it be maybe better to do it with the new v5.0 for Metasys 12?

**Comment by michaelgwelch** [comment]
Yes, I didn't meant to suggest we need to do it right now. Waiting until 12.0 is probably best. And when we rename it we should probably move it to the [Johnson Controls](https://github.com/johnsoncontrols) org.

We can leave this one behind in an archived state, with the README updated to point to new location.
